openlcsd/flow/tasks.py

```diff
@@ -969,8 +969,15 @@ def sync_result_to_corgi(context, engine):
     response = client.get("components", params={"uuid": component_uuid})
     response.raise_for_status()
```

Comment:
I think we should catch the exception for the response status

Reply:
What specific status do you think needs to deal with separately?

Reply:
Such as:
http://docs.python-requests.org/en/master/user/quickstart/#response-status-codes
https://github.com/RedHatProductSecurity/openlcs/blob/94c7cbb03af303ba902ee52018c917edacc0a2d7/openlcsd/flow/tasks.py#L951

Reply:
Handling exception case by case is useful especially when it's in a db transaction. But for this specific case, I think it's more verose to raise it directly insteading of adding any additional wrapper.

Reply:
Got it, thanks for your explanation.